src/Storage.Interface/Models/ShadowFields.cs

```diff
@@ -0,0 +1,25 @@
+ using Newtonsoft.Json;
+
+namespace Altinn.Platform.Storage.Interface.Models
+{
+    /// <summary>
+    /// Represents an object with information about how shadow fields are configured for the data type.
+    /// </summary>
+    [JsonObject(ItemNullValueHandling = NullValueHandling.Ignore)]
+    public class ShadowFields
+    {
+        /// <summary>
+        /// Gets or sets the prefix to use to filter out shadow fields.
+        /// </summary>
+        [JsonProperty(PropertyName = "prefix")]
+        public string Prefix { get; set; }
+
+        /// <summary>
+        /// Gets or sets the data type to save filtered data (without shadow fields) to.
+        /// Optional. If not set, the containing data type will be updated.
+        /// </summary>
+        [JsonProperty(PropertyName = "saveToDataType")]
+        public string SaveToDataType { get; set; }
```

Comment:
Wondering why we need a reference to the dataType withing the shadowField object if it is already a sub-property of a given data type?

Reply:
The property is meant to specify the data type to save the shadow fields to - i.e. a different data type. If specified, we will use this dataType and save the updated data to that dataType. If not specified, we use the containing data type (the one that this config is set for)

Reply:
Would something like `DataTypeForFilteredData` be clearer?

Reply:
I think we can leave it as is. Based on your